homeassistant/components/binary_sensor/zha.py

```diff
@@ -65,27 +65,24 @@
 async def _async_setup_remote(hass, config, async_add_entities,
                               discovery_info):
 
-    async def safe(coro):
-        """Run coro, catching ZigBee delivery errors, and ignoring them."""
-        import zigpy.exceptions
-        try:
-            await coro
-        except zigpy.exceptions.DeliveryError as exc:
-            _LOGGER.warning("Ignoring error during setup: %s", exc)
+    sensor = Switch(**discovery_info)
```

Comment:
sensor = switch ?

Reply:
Oops. I should code less late nights :) i'll fix it